### PR TITLE
Add 'release/trigger_all_tests.json' to trigger path of mass comment PR

### PR DESCRIPTION
**Pull request by Abacn**
This replaces the mass comment workflow for release verification. Currently add all precommit to it.

* Create GItHub Action PreCommit

* Create missing Google-ads IO PreCommit

* Create missing RequestResponse IO PreCommit

**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

File: .github/workflows/beam_PreCommit_GHA.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        github_job: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+    - name: Setup environment
+      uses: ./.github/actions/setup-environment-action
+      with:
+        java-version: 8
+        go-version: 1.21
+    - name: run goPreCommit script
+      uses: ./.github/actions/gradle-command-self-hosted-action
+      with:
+        gradle-command: :beam-test-gha:preCommit
```
Comment (Abacn) [inline .github/workflows/beam_PreCommit_GHA.yml:89]: `./gradlew :beam-test-gha:preCommit` passed locally

**Comment by Abacn** [comment]
R: @damccorm

**Comment by Abacn** [comment]
R: @volatilemolotov

**Comment by volatilemolotov** [comment]
Code looks good. 
With this trigger it will get added to PR so the status will show

**Comment by Abacn** [comment]
> Code looks good. With this trigger it will get added to PR so the status will show

Yeah that's the main benefits

- the down side is that we need to make sure this path is included in every test, so I added another precommit (PreCommit GHA) to guard this.

File: .github/workflows/beam_PreCommit_GHA.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      matrix:
+        job_name: [beam_PreCommit_GHA]
+        job_phrase: [Run GHA PreCommit]
+    timeout-minutes: 30
+    if: |
+      github.event_name == 'push' ||
+      github.event_name == 'pull_request_target' ||
+      github.event_name == 'schedule' ||
+      github.event_name == 'workflow_dispatch' ||
+      github.event.comment.body == 'Run Go PreCommit'
```
Comment (volatilemolotov) [inline .github/workflows/beam_PreCommit_GHA.yml:72]: This needs to be Run GHA PreCommit

**Review by volatilemolotov — COMMENTED** [review]
Just found a small issue with the conditional block for the job

File: .github/workflows/beam_PreCommit_GHA.yml (hunk 2 above)
Comment (Abacn) [reply .github/workflows/beam_PreCommit_GHA.yml]: Thanks, fixed, PTAL

**Review by damccorm — APPROVED** [review]
LGTM, thanks!

**Review by damccorm — APPROVED** [review]

**Pull request merged**